cmake/modules/FindSasl2.cmake

```diff
@@ -105,6 +105,8 @@ if(Sasl2_FOUND AND NOT TARGET Sasl2::Sasl2)
 endif()
 
 mark_as_advanced(Sasl2_LIBRARIES Sasl2_INCLUDE_DIRS Sasl2_VERSION)
+message("${Sasl2_LIBRARIES}")
+message("${Sasl2_INCLUDE_DIRS}")
```

Comment:
I guess this is for debugging only? ;-)

Reply:
Oh well, I didn't mean to commit this. This will be gone soon.